config/locales/en.yml

```
@@ -119,6 +119,7 @@ en:
     confirmation:
       title: "Registration complete"
       description: |
+        <p class="govuk-body">That’s everything - you will not receive an email or text message to confirm your registration.</p>
```

There was some talk of preferring en dashes earlier.

```suggestion
        <p class="govuk-body">That’s everything – you will not receive an email or text message to confirm your registration.</p>
```